template.tpl

```diff
@@ -723,9 +799,8 @@ scenarios: []
 
 
 ___NOTES___
-Cookiebot CMP Tag v2.2.2
```

Comment:
Why are we deleting this version comment?

Reply:
That's an oversight. I'll add that again